### DISCUSSION TITLE
NYI in the JVM

### DISCUSSION THREAD

**Discussion by coke** [perl6/doc#4136] [Ideas]
Found in the docs: "NOTE: These are not yet implemented in the JVM."

These kinds of notes are similar to the "requires a specific version of rakudo" note; we should add a note somewhere in the overview docs that we are not 100% consistent in mentioning these and treat them to the same formatting hypothesized in #302

**Comment by 2colours** [comment]
> (...) and treat them to the same formatting hypothesized in

So something like

```pod6
=for :lang<6.d+> :early-impl(Rakudo => 2018.06)
```

Am I understanding it right?
